src/components/organisms/DashboardPane/MenuItem.tsx

```diff
@@ -0,0 +1,138 @@
+import {useCallback, useEffect, useRef, useState} from 'react';
+
+import {FundProjectionScreenOutlined} from '@ant-design/icons';
+
+import {setActiveDashboardMenu, setDashboardSelectedResourceId} from '@redux/dashboard';
+import {useAppDispatch, useAppSelector} from '@redux/hooks';
+import {useResourceMetaMap} from '@redux/selectors/resourceMapSelectors';
+import {problemsSelector, useValidationSelector} from '@redux/validation/validation.selectors';
+
+import {DashboardMenu} from '@shared/models/dashboard';
+import {ResourceMeta} from '@shared/models/k8sResource';
+import {trackEvent} from '@shared/utils';
+
+import {CLICKAKBLE_RESOURCE_GROUPS} from '../Dashboard';
+import {ErrorCell, Resource, Warning} from '../Dashboard/Tableview/TableCells.styled';
+import * as S from './DashboardPane.style';
+
+export const MenuItem = ({
+  type,
+  menuItem,
+  onActiveMenuItem,
+}: {
+  type: 'parent' | 'child';
+  menuItem: DashboardMenu;
+  onActiveMenuItem?: Function;
+}) => {
+  const clusterResourceMeta = useResourceMetaMap('cluster');
+  const clusterConnectionOptions = useAppSelector(state => state.main.clusterConnectionOptions);
```

Comment:
We can use `useRefSelection` for this one since it's not used for rerendering UI

Reply:
Done!